lambda/src/main/resources/db/migration/V96__additional_metadata_statuses_backfill.sql

```diff
@@ -0,0 +1,30 @@
+-- Insert default 'NotEntered' value for 'ClosureMetadata' and 'DescriptiveMetadata' file statuses
+
+DO $$
+
+    DECLARE
+        closureMetadataFileIds uuid[] = ARRAY(
+            SELECT DISTINCT "FileId" FROM "FileStatus" WHERE NOT EXISTS (SELECT * FROM "FileStatus" WHERE "StatusType" = 'ClosureMetadata')
```

Comment:
If I run this on integration, I don't get any results because there are 'ClosureMetadata' entries in the `FileStatus` table. Is this right? 

Should it be something like `SELECT DISTINCT "FileId" FROM "FileStatus" WHERE "FileId" NOT IN (SELECT "FileId" FROM "FileStatus" WHERE "StatusType" = 'ClosureMetadata');` 
Or will this work because prod doesn't have and `ClosureMetadata` rows?

Reply:
> _If I run this on integration, I don't get any results because there are 'ClosureMetadata' entries in the FileStatus table. Is this right?_

No that's not right, it should be pretty much every file getting the statuses added, as these have only just been added as defaults. It worked locally, so something is not right. Need to take a look.

Reply:
@MancunianSam So yeah, you're selection logic is correct.